src/Shared/TypeLoader.cs

```diff
@@ -189,7 +189,8 @@ private static Assembly LoadAssembly(AssemblyLoadInfo assemblyLoadInfo)
                     // - otherwise, the assembly from the user specified path is loaded, if it exists.
                      
                     var assemblyNameInExecutableDirectory = Path.Combine(BuildEnvironmentHelper.Instance.CurrentMSBuildToolsDirectory,
-                        Path.GetFileName(assemblyLoadInfo.AssemblyFile));
+
+                    Path.GetFileName(assemblyLoadInfo.AssemblyFile));
```

Comment:
Yes, and the second argument to Combine is not indented, making it look like a separate statement, which makes the Path.Combine look like invalid syntax.